Assets/Scripts/SpriteController.cs

```diff
@@ -14,8 +14,12 @@ private void Awake()
 
 	public void MoveAnimation(int index)
     {
-        parentAnim.SetInteger("Checkpoint", index);
-		childAnim.SetBool ("isIdling", true);
+		if (parentAnim.enabled == true) 
+		{
+			parentAnim.SetInteger("Checkpoint", index);
+			childAnim.SetBool ("isIdling", true);
+		}
+        
 		/*switch (index)
```

Comment:
take this block comment out since it was used for disabling the animator in the lake